templates/deployment.yaml

```diff
@@ -57,7 +57,18 @@ spec:
           {{- toYaml .Values.dcEngineSecurityContext | nindent 10 }}
         volumeMounts:
         - name: dataconnect-engine-vol
-          mountPath: /opt/Actian/engine
+          mountPath: /opt/Actian/engine 
+      {{- if .Values.logstream.enabled }}    
+      - name: init-enginelog
+        image: {{ .Values.busybox.image.repository}}:{{ .Values.busybox.image.tag}}
+          imagePullPolicy: {{ .Values.busybox.image.pullPolicy }}
+          command: ['sh', '-c', 'chown -R 1000:2000 /opt']
```

Comment:
Why chown on /opt instead of /opt/Actian/local/work/job?
And I would use dcEngineSecurityContext instead of podSecurityContext since podSecurityContext will have more restrictions that may prevent you from performing the chown